libs/sdk-ui-filters/src/AttributeFilter@next/Components/Addons/AttributeDisplayFormSelect/AttributeDisplayFormDropdownButton.tsx

```diff
@@ -0,0 +1,52 @@
+// (C) 2021-2022 GoodData Corporation
+
+import React from "react";
+import classNames from "classnames";
+import { WrappedComponentProps, injectIntl } from "react-intl";
+import { Button } from "@gooddata/sdk-ui-kit";
+
+/**
+ * @internal
+ */
+export interface IToggleButtonProps {
+    text: string;
+    isOpened: boolean;
+    toggleDropdown: (e: React.SyntheticEvent) => void;
+}
+
+/**
+ * @internal
+ */
+export type ToggleButtonProps = IToggleButtonProps & WrappedComponentProps;
+
+const WrappedAttributeDisplayFormButton: React.FC<ToggleButtonProps> = ({
+    isOpened,
+    text: title,
+    toggleDropdown,
+    intl,
+}) => {
+    const buttonClassNames = classNames(
+        "gd-button-primary",
+        "gd-button-small",
+        "button-dropdown",
+        "dropdown-button",
+        "gd-attribute-filter-display-form-button",
+        "s-attribute-display-form-button",
+        {
+            "is-active": isOpened,
+        },
+    );
+    const iconRight = isOpened ? "gd-icon-navigateup" : "gd-icon-navigatedown";
+    const buttonTitle = intl.formatMessage({ id: "attributesDropdown.display_as" }, { title });
+    return (
+        <Button
+            className={buttonClassNames}
+            title={buttonTitle}
+            value={buttonTitle}
+            onClick={toggleDropdown}
+            iconRight={iconRight}
+        />
+    );
+};
+
+export const AttributeDisplayFormButton = injectIntl(WrappedAttributeDisplayFormButton);
```

Comment:
Please replace this with `useIntl()` so we don't pollute the props with intl property + we can get rid of `Wrapped` component

Reply:
done